features/perpetual/contract-state/GeneralInfo.tsx

```diff
@@ -32,160 +31,128 @@ const Status = styled(Typography)`
+                ((url.includes("coinbase") && "Coinbase") ||
+                  (url.includes("kraken") && "Kraken") ||
+                  (url.includes("binance") && "Binance") ||
+                  (url.includes("bitstamp") && "Bitstamp") ||
+                  "") +
+                (index < sourceUrls.length - 1 ? ", " : "]")}
             </Link>
-            ){`: `}
-          </Label>
-          {isExpired}
-        </Status>
-
-        <Status>
-          <Label>Price identifier: </Label>
-          {priceIdUtf8}
-        </Status>
-
-        <Status>
-          <Label>Identifier price: </Label>
-          {`${prettyLatestPrice}`}
-        </Status>
-
-        <Status>
-          <Label>Identifier sources: </Label>
-          {sourceUrls.map((url: string, index: number) => {
-            return (
-              <Link
-                key={index}
-                href={url}
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                {(index === 0 ? " [" : "") +
-                  ((url.includes("coinbase") && "Coinbase") ||
-                    (url.includes("kraken") && "Kraken") ||
-                    (url.includes("binance") && "Binance") ||
-                    (url.includes("bitstamp") && "Bitstamp") ||
-                    "") +
-                  (index < sourceUrls.length - 1 ? ", " : "]")}
-              </Link>
-            );
-          })}
-        </Status>
-        <Status>
-          <Label>Global collateral ratio: </Label>
-          <Tooltip
-            title={`The Global Collateralization Ratio (GCR) is the ratio of the total amount of collateral to total number of outstanding tokens.`}
-          >
-            <span>{pricedGcr}</span>
-          </Tooltip>
-        </Status>
-        <Status>
-          <Label>Collateral requirement: </Label>
-          {collReqPct}
-        </Status>
-        <Status>
-          <Label>Unique sponsors: </Label>
-          {sponsorCount}
-        </Status>
-        <Status>
-          <Label>Minimum sponsor tokens: </Label>
-          {minSponsorTokensSymbol}
-        </Status>
-      </Box>
-    );
-  }
+          );
+        })}
+      </Status>
+      <Status>
+        <Label>Global collateral ratio: </Label>
+        <Tooltip
+          title={`The Global Collateralization Ratio (GCR) is the ratio of the total amount of collateral to total number of outstanding tokens.`}
+        >
+          <span>{pricedGcr}</span>
+        </Tooltip>
+      </Status>
+      <Status>
+        <Label>Collateral requirement: </Label>
+        {collReqPct}
+      </Status>
+      <Status>
+        <Label>Unique sponsors: </Label>
+        {sponsorCount}
+      </Status>
+      <Status>
+        <Label>Minimum sponsor tokens: </Label>
+        {minSponsorTokensSymbol}
+      </Status>
+    </Box>
+  );
+}
+
+// TODO: price feed will need to be updated to use uniswap and also be able to calculate funding rate
```

Comment:
I wonder if we can export some of the pricefeed logic from @uma/financial-templates-lib, otherwise we're just gonna end up re writing everything.

Reply:
that is the dream but i dont think we are quite there yet